docs/format.rst

```diff
@@ -12,20 +12,82 @@ A `Lance Dataset` is organized in a directory.
         data/*.lance  -- Data directory
         latest.manifest -- The manifest file for the latest version.
         _versions/*.manifest -- Manifest file for each dataset version.
+        _fragment_manifests/*.lance -- Lance files containing data file metadata.
         _indices/{UUID-*}/index.idx -- Secondary index, each index per directory.
 
+.. image:: _static/format_overview.png
 
 A ``Manifest`` file includes the metadata to describe a version of the dataset.
+The manifest file references one or more fragment manifests. The fragment manifests
+are Lance files where each row is a fragment, and contains the path, metadata,
+and statistics for those fragments.
+
+A fragment is a collection of rows. It is made up of multiple Lance files,
+which collectively contain all the columns in the fragment. It also may contain
+a deletion file, which houses the row ids of deleted rows.
+
+Manifest format
+~~~~~~~~~~~~~~~
+
+The manifest file format is defined by a protobuf message:
 
 .. literalinclude:: ../protos/format.proto
    :language: protobuf
    :linenos:
    :start-at: // Manifest is
    :end-at: } // Manifest
 
-``DataFragment`` represents a chunk of data in the dataset. Itself includes one or more ``DataFile``,
-where each ``DataFile`` can contain several columns in the chunk of data. It also may include a 
-``DeletionFile``, which is explained in a later section.
+Fragment manifest format
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The fragment manifest is a Lance file, where each row is a fragment. The minimal schema
+is:
+
+.. code-block::
+
+    id: u32
+    data_files: struct
+        path: string
+        fields: list<u32>
+    deletion_file: struct
```

Comment:
You mean each row is a `arrow row`?

Reply:
No, I meant this to be a PARENT column in Lance. So `deletion_file.file_type` is its own column.